### PR TITLE
[doc] [dune] Update Dune build instructions

### PR DESCRIPTION
**Pull request by ejgallego**
TODO before 8.12:

- include composition
- include native

**Kind:** documentation

### CODE REVIEW

File: doc/sphinx/practical-tools/utilities.rst [hunk 1]
```diff
@@ -503,32 +503,38 @@ To build, say, two targets foo.vo and bar.vo in parallel one can use
...
      (name <module_prefix>)
-     (public_name <package.lib_name>)
+     (package <opam_package>)
      (synopsis <text>)
      (modules <ordered_set_lang>)
      (libraries <ocaml_libraries>)
      (flags <coq_flags>))
 
 This stanza will build all `.v` files in the given directory, wrapping
 the library under ``<module_prefix>``. If you declare a
```
Comment (Zimmi48) [inline doc/sphinx/practical-tools/utilities.rst]:
```suggestion
the library under ``<module_prefix>``. If you declare an
```

File: doc/sphinx/practical-tools/utilities.rst [hunk 2]
```diff
@@ -503,32 +503,38 @@ To build, say, two targets foo.vo and bar.vo in parallel one can use
...
-``<package.lib_name>`` a ``.install`` file for the library will be
+``<opam_package>`` a ``.install`` file for the library will be
```
Comment (Zimmi48) [inline doc/sphinx/practical-tools/utilities.rst]:
```suggestion
``<opam_package>``, an ``.install`` file for the library will be
```

File: doc/sphinx/practical-tools/utilities.rst [hunk 3]
```diff
@@ -503,32 +503,38 @@ To build, say, two targets foo.vo and bar.vo in parallel one can use
...
 generated; the optional ``<modules>`` field allows you to filter
```
Comment (Zimmi48) [inline doc/sphinx/practical-tools/utilities.rst]:
```suggestion
generated; the optional ``(modules <ordered_set_lang>)`` field allows you to filter
```

or

```suggestion
generated; the optional ``modules`` field allows you to filter
```

File: doc/sphinx/practical-tools/utilities.rst [hunk 4]
```diff
@@ -503,32 +503,38 @@ To build, say, two targets foo.vo and bar.vo in parallel one can use
...
 the list of modules, and ``<libraries>`` allows to depend on ML
```
Comment (Zimmi48) [inline doc/sphinx/practical-tools/utilities.rst]:
```suggestion
the list of modules, and ``(libraries <ocaml_libraries>)`` allows to depend on ML
```

or

```suggestion
the list of modules, and ``libraries`` allows to depend on ML
```

**Review by Zimmi48 — APPROVED** [review]
Great, thanks! Some form comments.

**Comment by Zimmi48** [comment]
No reason not to backport this, right?

**Comment by ejgallego** [comment]
Thanks a lot for the comments, updated.

> No reason not to backport this, right?

Indeed this can be backported IMO.

**Pull request merged**